Ghidra/Processors/8048/data/manuals/8048.idx

```diff
@@ -0,0 +1,54 @@
+@8048.pdf [MCS-48 Microcomputer User's Manual, February 1978]
```

Comment:
Do you have the exact pdf you used for the index.  The ones I'm finding aren't matching up.

Reply:
Should I put the [Vintage Technology Association URL](http://www.decadecounter.com/vta/pdf/mcs48.pdf) here?

Reply:
I wouldn't use a hard url that could change.  I've found lots of scans of the full manual from INTEL.  If you can, provide the most general description string in the index file that will allow someone to find the particular indexed version of the manual if they search for it.  If there is only one place and one version that matches your index, it might be worth re-indexing to another more generally available copy.

Reply:
Well, the thing is that it's not different versions of the manual, it's just different scans of _the same_ manual.  Some include the cover in the scan, some do not, some add a cover page of their down, some do not, etc.  The only way to make the index correspond reliably to a certain version of the manual (as opposed to a certain PDF containing that version of the manual) is to use the page numbers printed on the paper pages, but then the index is useless for automatically opening the correct page since none of the PDFs have those page numbers in their internal index...